### ISSUE TITLE
[OSX] TimeoutError: driver.quit()

### ISSUE THREAD

**Issue by srichards2103** [kaliiiiiiiiii/Selenium-Driverless#54]
TimeoutError on MacOS when quitting driver: 

```
Traceback (most recent call last):
  File "/Users/srichards/.pyenv/versions/3.11.1/lib/python3.11/site-packages/selenium_driverless/webdriver.py", line 434, in quit
    check_timeout(start, timeout)
  File "/Users/srichards/.pyenv/versions/3.11.1/lib/python3.11/site-packages/selenium_driverless/utils/utils.py", line 119, in check_timeout
    raise TimeoutError(f"driver.quit took longer than timeout: {timeout}")
TimeoutError: driver.quit took longer than timeout: 30
```

with this code: 
```python
from selenium_driverless import webdriver

async with webdriver.Chrome() as driver:
    await driver.get('https://www.google.com')
    await driver.sleep(0.5)
    await driver.quit()
```

**Comment by kaliiiiiiiiii** [comment]
that's at https://github.com/kaliiiiiiiiii/Selenium-Driverless/blob/93cb59a6590bbe07e5ea727ec8ebed466b791fd6/src/selenium_driverless/webdriver.py#L434

@srichards2103 Did the Browser-window close?

**Comment by srichards2103** [comment]
@kaliiiiiiiiii yeah the browser window closes, then it goes on to timeout after 30 seconds.

**Comment by kaliiiiiiiiii** [comment]
@srichards2103 Oh that's interesting. So maybe MacOs doesn't trow an OSError lol

**Comment by srichards2103** [comment]
Just tried by changing `os.kill` to `os.killpg` as suggested in that thread. Seems to work no problem.

**Comment by kaliiiiiiiiii** [comment]
> @kaliiiiiiiiii Just tried by changing `os.kill` to `os.killpg` as suggested in that thread. Seems to work no problem.

Thank's a lot for testing! I'll test if that works on Windows and Linux as well as expected & implement it.

**Comment by kaliiiiiiiiii** [comment]
@srichards2103 
This should be resolved with [`selenium-driverless==1.6`](https://github.com/kaliiiiiiiiii/Selenium-Driverless/discussions/75)
Please close this issue if it's fixed now:)
